examples/cluster/plot_face_segmentation.py

```diff
@@ -26,26 +26,30 @@
 
 import numpy as np
 import scipy as sp
+from scipy.ndimage.filters import gaussian_filter
 import matplotlib.pyplot as plt
+from skimage.transform import rescale
 
 from sklearn.feature_extraction import image
 from sklearn.cluster import spectral_clustering
-from sklearn.externals._pilutil import imresize
 
 
 # load the raccoon face as a numpy array
 try:  # SciPy >= 0.16 have face in misc
     from scipy.misc import face
-    face = face(gray=True)
+    orig_face = face(gray=True)
 except ImportError:
-    face = sp.face(gray=True)
+    orig_face = sp.face(gray=True)
 
 # Resize it to 10% of the original size to speed up the processing
-face = imresize(face, 0.10) / 255.
+# Applying a Gaussian filter for smoothing prior to down-scaling
+# reduces aliasing artifacts.
+smoothened_face = gaussian_filter(orig_face/255., sigma=(1/0.10-1)/2.)
```

Comment:
Actually you don't need that:

http://scikit-image.org/docs/dev/api/skimage.transform.html#skimage.transform.rescale

You can turn `anti_aliasing=True` to get the same effect.

Reply:
Actually, this is only in the current dev-version. In the [docs of the current stable version](http://scikit-image.org/docs/0.13.x/api/skimage.transform.html#skimage.transform.rescale) (0.13.1) it is not listed, yet. One might want to keep this in mind and change in the (rather far) future, though.
([See also discussion here](https://github.com/scikit-learn/scikit-learn/pull/10502#issuecomment-359487323))

Reply:
Agreed. In this case rewrite it as:

```python
smoothened_face = gaussian_filter(orig_face, sigma=4.5)
```